kubeflow/gcp/iap.libsonnet

```diff
@@ -9,6 +9,7 @@
       envoyAdminPort: 8001,
       envoyStatsPort: 8025,
       useIstio: util.toBool(_params.useIstio),
+      ingressService: "envoy-ingress"
```

Comment:
can we call this ingressName? I think it's more clear

Reply:
Please also change the ingress component name to use this variable

Reply:
done.